PiView_AG/Network.py

```diff
@@ -27,18 +27,19 @@ def host_name(self):
         # return platform.uname()[1]
         return gethostname()
 
-    def eth_name(self, type=None):
+    @staticmethod
+    def eth_name(_type=None):
```

Comment:
A bit pedantic on my part, but I don't like it when variable names shadow external names. Hence the `_type`.

Reply:
yeah, I tend to name them something like "ethernet_type"